frontend/src/authConfig.tsx

```diff
@@ -0,0 +1,21 @@
+export const msalConfig = {
+    auth: {
+        clientId: "f5993820-b7e2-4791-886f-f9f5027dc7be",
+        authority: "https://login.microsoftonline.com/3aa4a235-b6e2-48d5-9195-7fcf05b459b0",
+        redirectUri: "http://localhost:3001"
+    },
+    cache: {
+        cacheLocation: "sessionStorage"
+    }
+};
+
+export const loginRequest = {
+    // scopes: ["user.read"]
+    // scopes: ["api://" + msalConfig.auth.clientId + "/user_impersonation"]
+    scopes: ["api://f5993820-b7e2-4791-886f-f9f5027dc7be/user_impersonation"]
+}
+
+export const graphConfig = {
+    backendAPIEndpoint: "http://localhost:8000/robots",
+    originList: "http://localhost:3001"
```

Comment:
The base URL's should be in the config (`http://localhost:3001`), because they will be changed when we deploy the app